src/config/groups/members.cpp

```diff
@@ -20,16 +20,21 @@ std::optional<member> Members::get(std::string_view pubkey_hex) const {
     if (!info_dict)
         return std::nullopt;
 
-    auto result = std::make_optional<member>(std::string{pubkey_hex});
-    result->load(*info_dict);
+    auto sid = std::string{pubkey_hex};
+    auto result = std::make_optional<member>(sid);
+    auto is_pending_send = (pending_send_ids && pending_send_ids->find(sid) != pending_send_ids->end());
+    result->load(*info_dict, {{"local_pending_send", (is_pending_send ? 1 : 0)}});
```

Comment:
I'm thinking that it might be easier to just expose the `pending_send_ids` on the members object (i.e. make it public) so that calling can just query it directly.  Or, alternatively, just provide some simple wrappers to query and set it:
```C++
bool Members::has_pending_send(std::string sessionid) { return pending_send_ids.count(sessionid); }
void Members::set_pending_send(std::string sessionid, bool pending) {
    if (pending)
        pending_send_ids.insert(sessionid);
    else
        pending_send_ids.erase(sessionid);
}
```

Otherwise it feels a bit like we're shoehorning it through the `member` object though it feels a bit out of place there (and would seem to make things more complicated).

(We might have to revisit that if we end up with lots of things like this, but so far it feels like a one-off).

Reply:
I've made those fixes and pushed to Morgan's branch